.travis.yml

```diff
@@ -5,9 +5,10 @@ language: scala
 scala:
   - 2.11.11
   - 2.12.4
+  - 2.13.1
 
 jdk:
-  - oraclejdk8
+  - oraclejdk9
```

Comment:
Didn't licensing things change in a weird way with Oracle JDK9? To be safer we might want to stick to jdk8 and install the openjdk distributed by adoptopenjdk. There's an example at https://github.com/akka/akka/blob/master/.travis.yml#L10

Reply:
that's right, better switch to openjdk.
I switched to some jdk 9 since the oracle jdk8 was no longer supported, I guess we can switch to `openjdk8` instead.

Reply:
@raboof the jdk is now changed to `openjdk8`